.github/scripts/ci/check_outdated_tags.sh

```diff
@@ -0,0 +1,64 @@
+MASK='---\n(.*\n)*?((outdated: |outdated_since: ).+\n)(.*\n)*?---'
+PULL_REQUEST_TAG='SKIP_OUTDATED_CHECK'
+
+FIRST_COMMIT_HASH=$1
+LAST_COMMIT_HASH=$2
+FIRST_PR_COMMIT_HASH=$3
+
+# https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit
+function echo_red () { echo -e "\e[0;31m$1\e[m"; }
```

Comment:
`printf` could be a fitting replacement: 
![image](https://user-images.githubusercontent.com/4686101/165077166-63e96f65-4a19-44f9-8529-6425e23597a7.png)

Reply:
(and, since some shells provide it as a builtin as well as `echo`, it makes sense to use `/usr/bin/printf`)

Reply:
i'd rather put `#!/bin/bash` at the top of the file than deal with that

Reply:
that wouldn't work under macOS as well due to `bash` being old
```sh
bash-3.2$ env echo "\e[0;31mred\e[m"
\e[0;31mred\e[m
bash-3.2$ echo "\e[0;31mred\e[m"
\e[0;31mred\e[m
```

(`echo` is v 1.18 2005/01/10 here)